### ISSUE TITLE
Disabled Style/CommentAnnotation doesn't work

### ISSUE THREAD

**Issue by lashd** [bbatsov/rubocop#2974]
I don't want to put comments into the top of my files. At the moment I get the following message:

> Missing top-level class documentation comment

Here's  the contents of my .rubocop.yml file:

```yaml
Style/CommentAnnotation:
  Enabled: false
```

## Expected behavior
I expect no error.

## Actual behavior
I still see the offence reported when running rubocop.

## Steps to reproduce the problem

1. Create a ruby file with no comment at the top of the class definition.
2. Configure .rubocop.yml as above

## RuboCop version
```
$ rubocop -V
0.38.0 (using Parser 2.3.0.6, running on ruby 2.1.6 x86_64-darwin14.0)
```

**Comment by bbatsov** [comment]
You've disabled the wrong cop... You wanted to disable `Style/Documentation`.

**Comment by lashd** [comment]
When I disable `Style/Documentation` it doesn't capture comments I've left in the body of code.

Is there a way to disable what the offence mentions specifically?
